modules/katamari/CHANGELOG.md

```diff
@@ -12,6 +12,7 @@ and this project adheres to [Semantic Versioning](http://semver.org/spec/v2.0.0.
 ## 9.0.0 - 2022-03-03
 
 ### Added
+- Added a `Repeatable` type to the `Singleton` module
```

Comment:
This should be in a new Unreleased section above the 9.0.0 version and also I think the version.txt needs a bump or the library.

Reply:
Since this is a new type I also bumped the version to `9.1.0-rc`. Is that correct ?

Reply:
No that's not correct... TinyMCE is managed by lerna. I'd suggest reading the contributing docs 😉 https://github.com/tinymce/tinymce/blob/develop/CONTRIBUTING.md#version-management

Reply:
So if something needs a minor/major bump we track that by adding the version change to `versions.txt` as Spocke mentioned. In this case though that's already been done because of previous changes so it can be skipped this time.

Reply:
And I should rollback the change in `package.json` right ?

Reply:
Yes, as noted in the contributing document the versions in the package.json should never be edited manually.